### DISCUSSION TITLE
当backend服务器不返回content-type,bfe会自动添加一个，这个怎么解决？

### DISCUSSION THREAD

**Discussion by lf2186** [bfenetworks/bfe#969] [Q&A (问题和回答)] [answered]
最近上线业务，碰到一个问题，
当backend的response header为：
< HTTP/1.1 200 OK
* Server Apache-Coyote/1.1 is not blacklisted
< Server: Apache-Coyote/1.1
< Access-Control-Allow-Origin: *
< Access-Control-Allow-Methods: POST, GET, OPTIONS, DELETE
< Access-Control-Max-Age: 3600
< Access-Control-Allow-Headers: x-requested-with
< Transfer-Encoding: chunked
< Date: Fri, 21 Jan 2022 08:34:20 GMT

经过bfe的调度，会变为：
< HTTP/1.1 200 OK
< Access-Control-Allow-Headers: x-requested-with
< Access-Control-Allow-Methods: POST, GET, OPTIONS, DELETE
< Access-Control-Allow-Origin: *
< Access-Control-Max-Age: 3600
< Date: Fri, 21 Jan 2022 08:31:15 GMT
* Server server is not blacklisted
< Server: server
< X-Write-By:server_global_2.0
**_< Content-Type: text/plain; charset=utf-8_**
< Transfer-Encoding: chunked

多了Content-Type: text/plain; charset=utf-8，会导致二进制数据流变为文本被浏览器解析，这个怎么让bfe不要添加？？

**Comment by daimg** [comment, marked as answer]
The behavior, appending "Content-Type" if not present, can not be disabled currently, but you may overwrite this header's value to expected one by mod_header .
